### ISSUE TITLE
OpenIddict v 2.0.1 doesn't persist authorization properties in db

### ISSUE THREAD

**Issue by GDreyV** [openiddict/openiddict-core#914]
It looks like function PopulateAsync doesn't really populate descriptor properties: https://github.com/openiddict/openiddict-core/blob/2.0.1/src/OpenIddict.Core/Managers/OpenIddictAuthorizationManager.cs#L990 so they are not persisted in database and can't be populated later. I'm not making PR as I might be wrong in my idea how does it suppose to work in general. In my case I need to persist and update related tokens for current user so I don't want to send them as part of the claims.
Related question on stackoverflow: https://stackoverflow.com/questions/60326649/custom-properties-within-token/60354443

**Comment by kevinchalet** [comment]
It’s the expected behavior and it’s indicated in the XML documentation: https://github.com/openiddict/openiddict-core/blob/2.0.1/src/OpenIddict.Abstractions/Descriptors/OpenIddictAuthorizationDescriptor.cs#L25

I’ll consider removing this property bag in 3.0 and encourage users to subclass the descriptor class like we do in OrchardCore.

**Comment by kevinchalet** [comment]
`OpenIddictAuthorizationDescriptor.Properties` was removed when decoupling OpenIddict from ASP.NET Core and I just merged https://github.com/openiddict/openiddict-core/pull/915 to get rid of `OpenIddictTokenDescriptor.Properties`.

It's worth noting that despite the fact `OpenIddictAuthorizationDescriptor`/`OpenIddictTokenDescriptor` both had a `Properties` bag, this property was different from the one present in the 4 OpenIddict entities. The bag on the descriptors was meant to allow flowing custom authentication properties from the controller up to the manager, where you needed to add custom code to map them to custom columns, while the JSON-oriented `Properties` is a way to store data without having to change the DB schema (which is necessary for some Orchard scenarios).

In 3.0, OpenIddict no longer uses `AuthenticationProperties`, which is an ASP.NET Core-specific type. All authentication properties used in 2.x have been replaced by private claims (claims prefixed with `oi_` to avoid collisions).
